server/services/licence/licenceStatus.ts

```diff
@@ -227,24 +231,28 @@ function getCaStageState(licence) {
 
 function getRiskManagementState(licence) {
   const riskManagement = licence.risk?.riskManagement
+  const riskManagementVersion = riskManagement?.version
   const riskManagementAnswer = riskManagement?.planningActions
-  const awaitingInformationAnswer = riskManagement?.awaitingInformation
+  const checksConsideredAnswer = riskManagement?.hasConsideredChecks
+  const awaitingInformationAnswer = riskManagement?.awaitingInformation || riskManagement?.awaitingOtherInformation
   const { proposedAddressSuitable } = riskManagement || {}
 
   return {
     riskManagementNeeded: riskManagementAnswer === 'Yes',
+    mandatoryAddressChecksNotCompleted: checksConsideredAnswer === 'No',
```

Comment:
I would have though that this would depend on version? 

That this should never be true for version 1.

Also would have thought that we would want to flag if they haven't answered the question either: `mandatoryAddressChecksNotCompleted = checksConsideredAnswer  != 'Yes'`

Reply:
Sorted in recent commit